x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/components/fleet_event_filters_card.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useMemo, useState, useEffect } from 'react';
+import { ApplicationStart, CoreStart } from 'kibana/public';
+import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  PackageCustomExtensionComponentProps,
+  pagePathGetters,
+} from '../../../../../../../../../fleet/public';
+import { useKibana } from '../../../../../../../../../../../src/plugins/kibana_react/public';
+import { getEventFiltersListPath } from '../../../../../../common/routing';
+import { GetExceptionSummaryResponse } from '../../../../../../../../common/endpoint/types';
+import { PLUGIN_ID as FLEET_PLUGIN_ID } from '../../../../../../../../../fleet/common';
+import { MANAGEMENT_APP_ID } from '../../../../../../common/constants';
+import { LinkWithIcon } from './link_with_icon';
+import { ExceptionItemsSummary } from './exception_items_summary';
+import { EventFiltersHttpService } from '../../../../../event_filters/service';
+
+export const FleetEventFiltersCard = memo<PackageCustomExtensionComponentProps>(({ pkgkey }) => {
+  const {
+    services: {
+      application: { getUrlForApp },
+      http,
+    },
+  } = useKibana<CoreStart & { application: ApplicationStart }>();
+
+  const [stats, setStats] = useState<GetExceptionSummaryResponse | undefined>();
+  const eventFiltersListUrlPath = getEventFiltersListPath();
+  const eventFiltersApi = useMemo(() => new EventFiltersHttpService(http), [http]);
+
+  useEffect(() => {
+    const fetchStats = async () => {
+      const summary = await eventFiltersApi.getSummary();
```

Comment:
I think it also would be nice to show an error toast to the user when something goes wrong:
```js
const { addError } = useAppToasts();

useEffect(() => {
  const fetchStats = async () => {
    try {
      const summary = await eventFiltersApi.getSummary();
      setStats(summary);
    } catch (error) {
      addError(error, { title: i18n.SUMMARY_FETCH_FAILURE });
    }
  };
  fetchStats();
}, [eventFiltersApi, addError]);
```